sympy/integrals/transforms.py

```diff
@@ -1137,6 +1190,607 @@ def sbs(expr):
+    # 6.21
+    (exp(b*t)*sin(a*t),
+     a/((s-b)**2+a**2),
+     S.true, b, dco),
+    # 6.22
+    (exp(b*t)*cos(a*t),
+     (s-b)/((s-b)**2+a**2),
+     S.true, b, dco),
+    # 7.1
+    (erf(a*t),
+     exp(s**2/(2*a)**2)*erfc(s/(2*a))/s,
+     a>0, S.Zero, dco),
+    # 7.2
+    (erf(sqrt(a*t)),
+     sqrt(a)/sqrt(s+a)/s,
+     a>0, S.Zero, dco),
+    # 7.3
+    (exp(a*t)*erf(sqrt(a*t)),
+     sqrt(a)/sqrt(s)/(s-a),
+     a>0, a, dco),
+    # 7.4
+    (erf(sqrt(a/t)/2),
+     (1-exp(-sqrt(a*s)))/s,
+     a>0, S.Zero, dco),
+    # 7.5
+    (erfc(sqrt(a*t)),
+     (sqrt(s+a)-sqrt(a))/sqrt(s+a)/s,
+     a>0, S.Zero, dco),
+    # 7.6
+    (exp(a*t)*erfc(sqrt(a*t)),
+     1/(s+sqrt(a*s)),
+     a>0, S.Zero, dco),
+    # 7.7
+    (erfc(sqrt(a/t)/2),
+     exp(-sqrt(a*s))/s,
+     a>0, S.Zero, dco),
+    # 8.1, 8.2
+    (besselj(n, a*t),
+     a**n/(sqrt(s**2+a**2)*(s+sqrt(s**2+a**2))**n),
+     And(a>0, n>-1), S.Zero, dco),
+    # 8.3, 8.4
+    (t**b*besselj(n, a*t),
+     2**n/sqrt(pi)*gamma(n+S.Half)*a**n*(s**2+a**2)**(-n-S.Half),
+     And(And(a>0, n>-S.Half), Eq(b, n)), S.Zero, dco),
+    # 8.5
+    (t**b*besselj(n, a*t),
+     2**(n+1)/sqrt(pi)*gamma(n+S(3)/2)*a**n*s*(s**2+a**2)**(-n-S(3)/2),
+     And(And(a>0, n>-1), Eq(b, n+1)), S.Zero, dco),
+    # 8.6
+    (besselj(0, 2*sqrt(a*t)),
+     exp(-a/s)/s,
+     a>0, S.Zero, dco),
+    # 8.7, 8.8
+    (t**(b)*besselj(n, 2*sqrt(a*t)),
+     a**(n/2)*s**(-n-1)*exp(-a/s),
+     And(And(a>0, n>-1), Eq(b, n*S.Half)), S.Zero, dco),
+    # 8.9
+    (besselj(0, a*sqrt(t**2+b*t)),
+     exp(b*s-b*sqrt(s**2+a**2))/sqrt(s**2+a**2),
+     b>0, S.Zero, dco),
+    # 8.10, 8.11
+    (besseli(n, a*t),
+     a**n/(sqrt(s**2-a**2)*(s+sqrt(s**2-a**2))**n),
+     And(a>0, n>-1), Abs(a), dco),
+    # 8.12
+    (t**b*besseli(n, a*t),
+     2**n/sqrt(pi)*gamma(n+S.Half)*a**n*(s**2-a**2)**(-n-S.Half),
+     And(And(a>0, n>-S.Half), Eq(b, n)), Abs(a), dco),
+    # 8.13
+    (t**b*besseli(n, a*t),
+     2**(n+1)/sqrt(pi)*gamma(n+S(3)/2)*a**n*s*(s**2-a**2)**(-n-S(3)/2),
+     And(And(a>0, n>-1), Eq(b, n+1)), Abs(a), dco),
+    # 8.15, 8.16
+    (t**(b)*besseli(n, 2*sqrt(a*t)),
+     a**(n/2)*s**(-n-1)*exp(a/s),
+     And(And(a>0, n>-1), Eq(b, n*S.Half)), S.Zero, dco),
+    # 8.17
+    (bessely(0, a*t),
+     -2/pi*asinh(s/a)/sqrt(s**2+a**2),
+     a>0, S.Zero, dco),
+    # 8.18
+    (besselk(0, a*t),
+     (log(s+sqrt(s**2-a**2)))/(sqrt(s**2-a**2)),
+     a>0, Abs(a), dco)
+    ]
+    return laplace_transform_rules
+
+def _laplace_cr(f, a, c, **hints):
+    """
+    Helper function that will return `(f, a, c)` unless `**hints` contains
+    `noconds=True`, in which case it will only return `f`.
+    """
+    conds = not hints.get('noconds', False)
+    if conds:
+        return f, a, c
+    else:
+        return f
+
+def _laplace_rule_timescale(f, t, s, doit=True, **hints):
+    """
+    This function tries to apply the time-scaling rule of the Laplace
+    transform and returns `None` if it cannot do it.
```

Comment:
This should explain what the "time-scaling" rule of the Laplace transform is and it would be good to have a doctest example.

Reply:
Also if this is an internal function rather than a function for end-users then I think it's good to state that explicitly in the docstring somehow such as "Internal helper function to do X"

Reply:
I thought the trailing underbar meant precisely that, but I will change it in all relevant docstrings.

Reply:
Yes, the leading underscore does indicate that it is internal.